src/pymovements/dataset/dataset.py

```diff
@@ -332,15 +301,9 @@ def pos2vel(
             Computation method. See :func:`~transforms.pos2vel()` for details, default: smooth.
         verbose : bool
             If True, show progress of computation.
-        **kwargs
+        **kwargs: Any
```

Comment:
Is pydoclint checking the spaces here?
Sometimes we use `parameter : type` sometimes `parameter: type`

Maybe we should be consistent with that?

But if we don't have a linter to support that this would be out of scope of this PR i think.

Reply:
it only checks if there is a type given not the correct format, I can check whether there is a linter for spaces.

I also think if we have strict `mypy` typing we do not necessarily need types within docstrings.